src/simtools/simtel/simulator_array.py

```diff
@@ -67,12 +67,13 @@ def _make_run_command(self, run_number=None, input_file=None):
         str
             Command to run sim_telarray.
         """
+        config_file_path = self.corsika_config.array_model.config_file_path
         self._log_file = self.get_file_name(file_type="log", run_number=run_number)
         histogram_file = self.get_file_name(file_type="histogram", run_number=run_number)
         output_file = self.get_file_name(file_type="output", run_number=run_number)
 
         command = str(self._simtel_path.joinpath("sim_telarray/bin/sim_telarray"))
-        command += f" -c {self.corsika_config.array_model.config_file_path}"
+        command += f" -c {config_file_path}"
```

Comment:
See comment above.

Reply:
Ah - correct and fixed (was used before; but now introduced config_file_dir.